engines/router/compose/kafka.yaml

```diff
@@ -23,6 +23,9 @@ services:
       - KAFKA_CFG_ADVERTISED_LISTENERS=CLIENT://kafka:9092,EXTERNAL://localhost:9093
       - KAFKA_CFG_AUTO_CREATE_TOPICS_ENABLE=true
       - KAFKA_INTER_BROKER_LISTENER_NAME=CLIENT
+      - KAFKA_CFG_MAX_REQUEST_SIZE=10485760
```

Comment:
Are all 3 values here required? Based on the changes to `engines/router/missionctl/log/resultlog/kafka.go`, I imagine we only need the second one. Can we clean up the extra ones?